src/types/index.d.ts

```diff
@@ -110,14 +110,33 @@ export interface TokenOverlay {
   mod: TokenMod
 }
 
+export type FileBuffer = number[]
```

Comment:
Why is it `number[]`. It's what you get after `FileReader.readAsArrayBuffer`? Then it's `ArrayBuffer` type

Reply:
done